concepts/80 Troubleshooting/45 Component Content is not Scrollable/05 Component Displays Only a Part of the Content.md

```diff
@@ -0,0 +1,8 @@
+Sometimes a component displays only a part of the content and a scrollbar does not appear. Check the following list for conditions of this situation:
+
+- One of the component ancestors in the DOM has a fixed height and `overflow: hidden`.
+- The component height is `auto`.
+
+For example, put DataGrid into Popup. The DataGrid [height](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#height) property's default value is `auto`. So, the DataGrid component automatically adjusts its height to accommodate all rows. The component does not need its own scrollbar because the height for rows is enough.
```

Comment:
I'm not sure this is correct. But I'm also not 100% sure what happens. If the grid needs to display 300 rows, is it going to extend the popup? If so, would the web page's scrollbar appear? Or nothing like that? 

```suggestion
For example, put a DataGrid into a Popup. The DataGrid's default [height](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#height) value is `auto`. This means that the component automatically adjusts its height to accommodate all rows and does not display a scrollbar. 
```